extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/utils/DateUtilities.java

```diff
@@ -47,4 +55,21 @@ public static Date parseISO(
 		return df.parse(input);
 
 	}
+
+	public static TemporalRange getTemporalRange(
+			final DataStorePluginOptions dataStorePlugin,
+			final ByteArrayId adapterId ) {
+		final DataStatisticsStore statisticsStore = dataStorePlugin.createDataStatisticsStore();
+		final CloseableIterator<DataStatistics<?>> statsIt = statisticsStore.getDataStatistics(adapterId);
+
+		while (statsIt.hasNext()) {
+			final DataStatistics stats = statsIt.next();
+			if (stats instanceof FeatureTimeRangeStatistics) {
```

Comment:
FeatureTimeRangeStatistics.composeId(<fieldName>) gets you the statistics ID that can then be used within that call to DataStatisticsStore